package.json

```diff
@@ -69,7 +70,7 @@
     ],
     "transformIgnorePatterns": [
       "/node_modules/(?!@open-formulieren/formio-builder)/",
-      "\\.pnp\\.[^\\\/]+$"
+      "\\.pnp\\.[^\\/]+$"
```

Comment:
uh this changes it from the pattern that's configured by default: https://jestjs.io/docs/configuration#transformignorepatterns-arraystring - I've no idea if this breaks something or not though.

Reply:
npm :point_left: did it.

Reply:
Had to turn off my formatter (jq) too, as it corrects it as well; "\/" is not a legal escape code in a string per se. Only inside a `/repattern/` literal it makes sense. So if I reason about it `/\.pnp\.[^\/]+$/` is probably the regex they're after, but this
```node
> JSON.stringify(/\.pnp\.[^\/]+$/.toString())
'"/\\\\.pnp\\\\.[^\\\\/]+$/"'
```
just made me happy about r"" strings in Python and revert the change.

Some people, when confronted with a problem think: "I know, I'll use regular expressions... and JavaScript." Now they have NaN problems.

No seriously:
```node
> /./ * 2
NaN
```

Reply:
I may also be wrong in how I copied it from the docs where a JS regex is expected, but all these levels of configuration make me :dizzy: